pkgs/by-name/pr/prisma-engines_7/setup-hook.sh

```diff
@@ -0,0 +1,2 @@
+export PRISMA_SCHEMA_ENGINE_BINARY="@out@/bin/schema-engine"
+export PRISMA_FMT_BINARY="@out@/bin/prisma-fmt"
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/472380/commits/91ad6aff95484cffede2dd90b7f2a02e0099e05f